data/competencies.yml

```diff
@@ -925,3 +925,15 @@
   level: senior1-to-senior2
   area: leadership
   domain: null
+
+
+- id: f173b7e9-1c46-4afd-a659-d264aa173759
+  summary: Contributes to inclusive organisational culture
+  examples:
+    - Membership of internal groups that promote diversity and inclusion (such as FT Embrace, FT Proud, Mental health first aiders)
+    - Organises inclusive social events for groups of teams
```

Comment:
Could we add something around vulnerability?, ie. 
- Shares their mistakes and related learnings to foster a growth mindset culture

Reply:
I really like this suggestion 👍 do we think this is more the kind of behaviour we expect from an S1 or an S2?

Reply:
I think as soon as they get line reports, this should be in their "skill set". I think it's important when mentoring others because people probably only see your achievements and not your failures. 

On that thought, would it be a good use of time to have - once a year - a P1 retro day where short presentations are given, maybe even by the people who triggered them if they are happy to do so? I'd love to participate in something like that!

Reply:
The issue with "as soon as they get line reports" is that this time differs across groups at the FT. We also don't explicitly require line management at any level ([the reasoning for this is documented here](https://engineering-progression.ft.com/docs/faq/#is-there-an-individual-contributor-route)). So if we're adding in an example around sharing mistakes then we'll need to pick a level that this applies at. I think earlier is better for me so I'm leaning towards the S1 competency. Thoughts?

Reply:
I agree with Tatiana - everyone with or without line management experience can partake in this (I need to get better at posting questions publicly and not being so ashamed that I secretly DM people for assistance...!)